### PR TITLE
css polishing

### PR DESCRIPTION
**Pull request by AsmaaELM**
Issue: https://scality.atlassian.net/browse/ZKUI-120

### CODE REVIEW

File: src/react/ui-elements/TableKeyValue2.tsx [hunk 1]
```diff
@@ -57,6 +57,10 @@ export const Groups = styled.div`
 
   max-width: 450px;
 `;
+export const ScrollArea = styled.div`
+  overflow: auto;
+  max-height: calc(100vh - 19rem);
```
Comment (ChengYanJin) [inline src/react/ui-elements/TableKeyValue2.tsx:62]: On my side, the Cancel and Save button are still not visible by default...

**Review by ChengYanJin — CHANGES_REQUESTED** [review]

File: src/react/ui-elements/TableKeyValue2.tsx (hunk 1 above)
Comment (AsmaaELM) [reply src/react/ui-elements/TableKeyValue2.tsx:62]: I just tested.  on my branch I get this result:
Screenshot 2022-03-30 at 19 15 03
And in development, we have this result : 
Screenshot 2022-03-30 at 19 15 48

Or  maybe Yanjin, you're talking about another screen  🤔 ?

Comment (AsmaaELM) [reply src/react/ui-elements/TableKeyValue2.tsx:62]: Here is the expiration edition after css modification:
Screenshot 2022-03-31 at 10 10 06

Comment (ChengYanJin) [reply src/react/ui-elements/TableKeyValue2.tsx:62]: Hi @AsmaaELM 
Do you think we can avoid the empty space on the right side of the screen? 
by increasing the width of the workflow table maybe?

**Review by ChengYanJin — APPROVED** [review]

**Pull request merged**